movie.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+class Movie < Item
+  attr_accessor :id, :name, :publish_date, :silet
+
```

Comment:
Use attr_reader for id, I shouldn't be changed from created object

Reply:
thanks, @Sanja969 am adding on the required changes as soon as possible